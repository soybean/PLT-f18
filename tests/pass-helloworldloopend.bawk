BEGIN {}
LOOP {
print("Hello loop!");  
}
END {
print("Hello end!");  
}
