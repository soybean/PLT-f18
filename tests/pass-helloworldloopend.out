Hello loop!
Hello loop!
Hello loop!
Hello loop!
Hello loop!
Hello loop!
Hello end!
